editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/maven/CamelProjectConfigurator.java

```diff
@@ -49,11 +57,11 @@
 
 public class CamelProjectConfigurator extends AbstractProjectConfigurator {
 
-	private static final String ARTFIFACT_ID_CAMEL_PREFIX = "camel-"; //$NON-NLS-1$
-	private static final String GROUP_ID_ORG_APACHE_CAMEL = "org.apache.camel"; //$NON-NLS-1$
-	private static final String WAR_PACKAGE = "WAR"; //$NON-NLS-1$
-	private static final String BUNDLE_PACKAGE = "BUNDLE"; //$NON-NLS-1$
-	private static final String JAR_PACKAGE = "JAR"; //$NON-NLS-1$
+	public static final String ARTFIFACT_ID_CAMEL_PREFIX = "camel-"; //$NON-NLS-1$
```

Comment:
I think that you don't need to put these as public (at least in the scope of this PR)